build.user.properties.template

```diff
@@ -0,0 +1,21 @@
+nativity.dir=C:/Users/sync/Dev/streem/liferay-nativity
+nativity.version=1.0.2
+ms.sdk.7.1.dir=C:/Program Files/Microsoft SDKs/Windows/v7.1
+framework.dir=C:/Windows/Microsoft.NET/Framework64/v4.0.30319
+
+context.menu.guid={EBEDF891-A73A-4B35-91BB-57E3820C4BBC}
+
+overlay.name.synced=BoxDesktopSyncedOverlay
+overlay.guid.synced={1DEAD8A6-8293-411D-8201-C32E2D6F5005}
+overlay.id.synced=1
+overlay.path.synced=C:/Users/sync/Dev/streem/app/assets/overlays/Synced.ico
```

Comment:
According to our discussion, all the icons will be renamed soon. I will keep you posted about what they will be called in the future.